.changeset/spotty-insects-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-proxy-backend': minor
+---
+
+Support for an `allowedEndpoints` option for `plugin-proxy`.
```

Comment:
A think that comes into my mind: you need to add the new property to the config schema

Reply:
Thanks! Adding that now. :)